selfdrive/ui/qt/maps/map.cc

```diff
@@ -470,8 +478,15 @@ void MapInstructions::updateInstructions(cereal::NavInstruction::Reader instruct
     } else if (straight) {
       fn += "turn_straight";
     }
+    fn +=  + ".png";
```

Comment:
I assume the lane arrows don't need a RHD version?

Reply:
this section do not append modifier, I've removed them :+1: